administrator/components/com_media/controllers/file.json.php

```diff
@@ -56,6 +56,7 @@ public function upload()
 		// Get some data from the request
 		$file   = $this->input->files->get('files', '', 'array');
 		$folder = $this->input->get('folder', '', 'path');
+		$canOverwrite = $this->input->getBool('allowovewrite', false);
```

Comment:
That is indeed what I am talking about, that is what the old code did. The user could choose if the cropped image would overwrite the existing one or be saved as a copy.

P.s. there is a typo in the option name. It says allowovewrite instead of allowoverwrite. Notice the r ;)